src/deployer-watch/deployer-watch.spec.js

```diff
@@ -0,0 +1,98 @@
+const {
+  TransactionEvent,
+  FindingType,
+  FindingSeverity,
+  Finding,
+} = require('forta-agent');
+
+const config = require('../../agent-config.json');
+const { handleTransaction } = require('./deployer-watch');
+
+// see deployer-watch.json for full whitelist
+// and contract-addresses.json for latest Deployer address
+const deployerAddress = '0x2bAaA41d155ad8a4126184950B31F50A1513cE25'.toLowerCase();
```

Comment:
Any reason not to pull these addresses from the JSON file?

Reply:
good call